extensions/azurePublish/src/components/azureProvisionDialog.tsx

```diff
@@ -659,140 +666,108 @@ export const AzureProvisionDialog: React.FC = () => {
-                    placeholder={formatMessage('Select one')}
-                    styles={{ root: { paddingBottom: '8px' } }}
-                    onChange={updateCurrentLocation}
-                  />
+          {formData.choice === 'create' && (
+            <form style={{ width: '100%' }}>
+              <Dropdown
+                ariaLabel={formatMessage(
+                  'The Azure AD directory includes the tenant’s users, groups, and apps and is used to perform identity and access management functions for tenant resources.'
                 )}
-                {currentConfig?.settings?.luis?.region && currentLocation !== currentLuisLocation && (
-                  <TextField
-                    disabled
-                    required
-                    defaultValue={currentConfig?.settings?.luis?.region}
-                    label={formatMessage('Region for Luis')}
-                    styles={{ root: { paddingBottom: '8px' } }}
-                    onRenderLabel={onRenderLabel}
-                  />
-                )}
-                {!currentConfig?.settings?.luis?.region && currentLocation !== currentLuisLocation && (
-                  <Dropdown
-                    required
-                    defaultSelectedKey={currentConfig?.settings?.luis?.region || currentLuisLocation}
-                    label={formatMessage('Region for Luis')}
-                    options={luisLocationsOption}
-                    placeholder={formatMessage('Select one')}
-                    onChange={updateLuisLocation}
-                  />
+                disabled={allTenants.length === 1 || currentConfig?.tenantId}
+                errorMessage={loginErrorMsg}
+                label={formatMessage('Azure Directory')}
+                options={allTenants.map((t) => ({ key: t.tenantId, text: t.displayName }))}
+                selectedKey={formData.tenantId}
+                styles={{ root: { paddingBottom: '8px' } }}
+                onChange={(_e, o) => {
+                  updateFormData('tenantId', o.key as string);
+                }}
+                onRenderLabel={onRenderLabel}
+              />
+              <Dropdown
+                required
+                ariaLabel={formatMessage('All resources in an Azure subscription are billed together')}
+                disabled={currentConfig?.subscriptionId}
+                errorMessage={subscriptionsErrorMessage}
+                label={formatMessage('Subscription')}
+                options={subscriptionOptions}
+                placeholder={formatMessage('Select one')}
+                selectedKey={formData.subscriptionId}
+                styles={{ root: { paddingBottom: '8px' } }}
+                onChange={(_e, o) => {
+                  updateFormData('subscriptionId', o.key as string);
+                }}
+                onRenderLabel={onRenderLabel}
+              />
+              <ResourceGroupPicker
+                disabled={currentConfig?.resourceGroup}
```

Comment:
I don't understand this value for disable - if there isn't already a resource group selected then the group picker is disabled?